packages/mdc-fab/_mixins.scss

```diff
@@ -56,6 +56,30 @@
   width: 100%;
 }
 
+@mixin mdc-fab-extended-padding($icon-padding, $label-padding) {
+  @include mdc-fab-extended-label-padding($label-padding);
+
+  // Offsets the outer label padding by deducting it from icon padding.
+  .mdc-fab__icon {
+    @include mdc-rtl-reflexive-property(
+      margin,
+      $mdc-fab-extended-icon-padding - $mdc-fab-extended-label-padding,
```

Comment:
The variable names throughout these mixins need to be updated to reference the mixin parameters, otherwise they will never actually customize anything.

We should ideally have an additional screenshot test page under `mdc-fab/mixins` to test these.